### PR TITLE
Add matrix version

### PR DESCRIPTION
**Pull request by rodo**
Add matrix version to run postgresql maintained version

### CODE REVIEW

**Comment by rodo** [comment]
You can check the tests results on https://github.com/rodo/pgtap-github-actions/actions/runs/8031383530

File: .github/workflows/pgtest.yml [hunk 1]
```diff
@@ -7,10 +7,13 @@ jobs:
   pgtap:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '14', '15', '16' ]
```
Comment (francois2metz) [inline .github/workflows/pgtest.yml]: Could you rename it something like: 
```suggestion
        pg-version: [ '14', '15', '16' ]
```

**Comment by francois2metz** [comment]
Thanks for your contribution!

**Comment by francois2metz** [comment]
Thanks!

**Pull request merged**